frontend/package.json

```diff
@@ -80,6 +80,7 @@
     "autoprefixer": "^10.4.13",
     "babel-jest": "^29.5.0",
     "babel-loader": "^8.3.0",
+    "esbuild": "^0.17.19",
```

Comment:
Don't we have a bundler yet? Please avoid other changes so that your PRs can get merged faster.

Reply:
That was added accidentally while resolving a local build issue.